src/Core/SettingsChangesHistory.cpp

```diff
@@ -69,6 +69,8 @@ const VersionToSettingsChangesMap & getSettingsChangesHistory()
         /// Note: please check if the key already exists to prevent duplicate entries.
         addSettingsChanges(settings_changes_history, "25.5",
         {
+            /// Release closed. Please use 25.6
+            {"geotoh3_argument_order", "lon_lat", "lat_lon", "A new setting for legacy behaviour to set lon and lat argument order"},
             {"geotoh3_lon_lat_input_order", true, false, "A new setting for legacy behaviour to set lon and lat order"},
```

Comment:
I missed removing this but fixing in the backport PR: https://github.com/ClickHouse/ClickHouse/pull/80664